src/actions/activity.js

```diff
@@ -3,6 +3,11 @@
 import { ACTIVITY_MESSAGES_LIMIT } from '../global/Constants';
 import { type ActivityLogMessage } from '../types/ActivityLogMessage';
 
+export const MESSAGE_ADDED = 'MESSAGE_ADDED';
```

Comment:
Please revert all changes on that file.

Reply:
I am not sure if we are good to remove as I did not add them.

Reply:
I mean revert your changes only, so that file has no differences with develop branch. As I understand, your changes here are not intentional.